docker/centos6/Dockerfile

```diff
@@ -239,6 +240,45 @@ RUN source /opt/rh/devtoolset-8/enable && \
     mkdir -p /opt/boringssl/lib && \
     cp crypto/libcrypto.a ssl/libssl.a /opt/boringssl/lib/
 
+# build/install wolfssl
+RUN source /opt/rh/devtoolset-8/enable
+    curl -Ls https://github.com/wolfSSL/wolfssl/archive/refs/tags/v5.2.0-stable.zip -o wolfssl.zip
+    cat <<EOF > wolfssl.asc
```

Comment:
IIRC, heredoc does not work correctly in a Dockerfile. You may have to use the a pattern like the `printf` command (near the end of the `devel` section of this file). It'd not pretty, but avoids an extra file and a `COPY` directive.